create-intermediate-certs.sh

```diff
@@ -0,0 +1,68 @@
+echo "*****************  Certs creation  *************************"
+
+function echo {
+  COLOR="\e[93m";
+  ENDCOLOR="\e[0m";
+  printf "$COLOR%b$ENDCOLOR\n" "$1";
+}
+
+export CERT_FOLDER="$(pwd)/certs"
+export DOMAIN="subca.acalustra.com"
+
+mkdir -p $CERT_FOLDER
+rm -rf $CERT_FOLDER/*
+cd $CERT_FOLDER
+
+echo "Certs creation on folder: $CERT_FOLDER"
+
+echo ">> SSL create CA cert"
+openssl genrsa -out rootCA.key 4096
+openssl req -batch -new -x509 -nodes -subj "/CN=root.ca" \
+  -extensions v3_ca \
+  -key rootCA.key -sha256 -days 1024 -out rootCA.pem
+
+echo ">> Intermediate  CA cert"
+openssl genrsa -out subCA.key 4096
+openssl req -batch -new -x509 -subj "/CN=sub.ca" -nodes \
+  -extensions v3_ca \
```

Comment:
I think this should be `openssl req -batch -new -x509 -subj "/CN=sub.ca" -nodes -extensions v3_intermediate_ca -key subCA.key -sha256 -days 1024 -out subCA.pem` 

As described here in the [docs](https://openssl-ca.readthedocs.io/en/latest/create-the-intermediate-pair.html#create-the-intermediate-certificate) if a proper Intermediate CA signing certificate is required then this extension should be used as we want to generate another signed certificate later (the leaf certificate)